README.md

```diff
@@ -168,17 +168,36 @@ sites.
 
 ### Enabling
 
-To enable the full request cache the `RequestCache` service needs to be added to the `RequestProcessor` as a filter.
+To enable the full request cache, the `RequestCacheMiddleware` needs to be applied and a `Global` config block needs to be created:
```

Comment:
Replaced `RequestCache` (based on SS3’s `RequestFilter` API) with a middleware-based implementation. It’s functionally equivalent